src/Logary/Extensions/LogManager.fs

```diff
@@ -0,0 +1,15 @@
+namespace Logary
+
+open Logary.Metric
+
+open System.Runtime.CompilerServices
+
+[<AutoOpen; Extension>]
+module LogManagerEx =
+  let defaultMetricRegistry = new MetricRegistry()
```

Comment:
I strongly feel this should go the way of how we initialise the other registry and have a pointer it from here; a projection from the singleton of the last-initialised logmanager. I've updated https://github.com/haf/expecto/blob/master/Expecto/Logging.fs#L366-L389 to do that; and I would like to replace our current clock+flyweights to use the way we do it in Expecto.

Reply:
`MetricRegistry` seems a litter bit different, even if people change logary's global registry by `setGlobals`, the `defaultMetricRegistry` should not changed by that registry's metricRegisty. That is why it use a `static member DefaultMetricRegistry = ...`, log message or metrics can occur before logary's registry‘s initialise. log message can be dropped that's why we have a nulllogger, but since metrics is lighter, we should preserve these metrics before registry's initialise. 

If we use `DVar` here, we need do some merges (the old metrics and the new metrics), which seems not easy, especially when user using these metric at same time.

Reply:
but you are right, it will be a good replace for our current clock+flyweights loggers. will change it in another branch

Reply:
```
  let enableCounterMetric (counterConf: Metric.BasicConf) (message: Message) =
    message |> setContext KnownLiterals.CounterMetricConfContextName counterConf
```
In fact, I don't recommend such a way to bind messages(prefer **remove this function**),  one should add a metric type separately instead of using this shortcut. Then no ship configuration in the log message.